templates/components/form/header.html.twig

```diff
@@ -133,8 +132,12 @@
                         {% endif %}
 
                         <input class="form-check-input" type="checkbox" name="is_recursive" value="1"
-                              {% if checked %}checked="checked"{% endif %}
+                              {% if item.isRecursive() %}checked="checked"{% endif %}
                               {% if disabled %}disabled="disabled"{% endif %} />
+                        {% if item is instanceof('CommonDBChild') and item.isNewItem() and item.isRecursive() %}
+                            {# Send value on hidden field to ensure creation will use inherited recursivity on CommonDBChild #}
+                            <input type="hidden" name="is_recursive" value="1" />
```

Comment:
Unless I miss something, this will end with 2 input with the same name, isn't it? I may be wrong, but in that case, we're not entirely sure which one will be retrieved.

Reply:
First one is `disabled`, so it will not be send in POST.